src/components/Media/InfoPanel.js

```diff
@@ -0,0 +1,62 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import React from "react";
+import { MdCheck, MdClose, MdDelete, MdPlayArrow, MdStar } from "react-icons/md";
+
+const useStyles = makeStyles((theme) => ({
+  line: {
+    display: "flex",
+  },
+  icon: {
+    fontSize: "1.5rem",
+  },
+  text: {
+    width: "95%",
+    paddingLeft: theme.spacing(3),
+    paddingBottom: theme.spacing(3),
+  },
+  divider: {
+    width: "2px",
+  },
+}));
+
+function InfoPanel() {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <div className={classes.line}>
+        <MdCheck className={classes.icon} />
+        <Typography className={classes.text}>
+          Mit Klick auf dieses Icon kannst du Filme, die du bis zum Ende gesehen hast, in die
```

Comment:
```suggestion
          Mit Klick auf dieses Icon kannst du Filme und Serien, die du bis zum Ende gesehen hast, in die
```
Ist überall nur Film. Würde ich noch auf beides ändern denke ich 😇